### PR TITLE
Fix sign in screen, add validation

### PR DESCRIPTION
**Pull request by Julia-Kovalchuk**

### CODE REVIEW

File: src/components/DraggableBottomSheet/index.tsx [hunk 1]
```diff
@@ -9,10 +9,15 @@ import {
 import {StyleSheet, View, Animated} from 'react-native';
 import {Color} from '@theme/colors';
 import {Header} from '../Header';
+import {FormSignIn} from '../FormSignIn';
 
+const isAuth = false;
```
Comment (antoniosk10) [inline src/components/DraggableBottomSheet/index.tsx:14]: is it temporary decision?

File: src/components/DraggableBottomSheet/index.tsx [hunk 2]
```diff
@@ -83,7 +88,7 @@ export const DraggableBottomSheet = () => {
           <View style={styles.draggableArea} {...panResponder.panHandlers}>
             <View style={styles.dragHandle} />
           </View>
-          <Header isOpen={isOpen} />
+          {isAuth ? <Header isOpen={isOpen} /> : <FormSignIn isOpen={isOpen} />}
```
Comment (antoniosk10) [inline src/components/DraggableBottomSheet/index.tsx:91]: do you still need `isOpen`?

File: src/components/FormSignIn/index.tsx [hunk 3]
```diff
@@ -1,46 +1,77 @@
...
+    };
+  });
+
+  const changeOpacity = () => {
+    isOpen
+      ? (opacityValue.value = withTiming(1, {duration: 1500}))
+      : (opacityValue.value = withTiming(0, {duration: 300}));
+  };
+
+  isOpen ? changeOpacity() : changeOpacity();
```
Comment (antoniosk10) [inline src/components/FormSignIn/index.tsx:40]: ??

File: src/components/FormSignIn/index.tsx [hunk 4]
```diff
@@ -1,46 +1,77 @@
...
 
   const signInValues: ISignInValues = {
     email: '',
     password: '',
   };
 
+  const SigninSchema = Yup.object().shape({
```
Comment (antoniosk10) [inline src/components/FormSignIn/index.tsx:47]: signinSchema

File: src/components/FormSignIn/index.tsx [hunk 5]
```diff
@@ -1,46 +1,77 @@
...
+            <ViewCenter>
+              <SmallTitle>Please sign in to continue</SmallTitle>
+            </ViewCenter>
             <BlockBox>
               <FormInput
                 placeholder="Enter email"
                 onChangeText={handleChange('email')}
                 value={values.email}
               />
+              <TextError>{errors.email}</TextError>
```
Comment (antoniosk10) [inline src/components/FormSignIn/index.tsx:74]: `errors.email && <TextError>{errors.email}</TextError>`

File: src/components/FormSignIn/index.tsx [hunk 6]
```diff
@@ -49,28 +80,17 @@ export const FormSignIn = () => {
                 value={values.password}
                 secureTextEntry={true}
               />
+              <TextError>{errors.password}</TextError>
```
Comment (antoniosk10) [inline src/components/FormSignIn/index.tsx:83]: errors. password && <TextError>{errors. password}</TextError>

File: src/components/ui/SmallButton/index.tsx [hunk 7]
```diff
@@ -1,14 +1,22 @@
...
+import {StyleSheet} from 'react-native';
+import {Box, ButtonContainer} from './styles';
 import {Color} from '@theme/colors';
 import {IButtonProps} from '@customTypes/SmallButton-types';
 import Ionicons from 'react-native-vector-icons/Ionicons';
 import MaterialIcons from 'react-native-vector-icons/MaterialIcons';
+import MaterialCommunityIcons from 'react-native-vector-icons/MaterialCommunityIcons';
 
 export const SmallButton = ({type, onPress, ...props}: IButtonProps) => {
+  const styles = StyleSheet.create({
```
Comment (antoniosk10) [inline src/components/ui/SmallButton/index.tsx:10]: useMemo

File: src/components/ui/SmallButton/index.tsx [hunk 8]
```diff
@@ -1,22 +1,36 @@
...
 import {Color} from '@theme/colors';
 import {IButtonProps} from '@customTypes/SmallButton-types';
 import Ionicons from 'react-native-vector-icons/Ionicons';
 import MaterialIcons from 'react-native-vector-icons/MaterialIcons';
+import MaterialCommunityIcons from 'react-native-vector-icons/MaterialCommunityIcons';
+import {useMemo} from 'react';
 
 export const SmallButton = ({type, onPress, ...props}: IButtonProps) => {
+  const containerStyles = useMemo(
+    () => [
```
Comment (antoniosk10) [inline src/components/ui/SmallButton/index.tsx:11]: we don't need array here

**Review by KhaninPasha0488 — APPROVED** [review]

**Review by antoniosk10 — APPROVED** [review]

**Pull request merged**